contrib/IECoreUSD/test/IECoreUSD/USDSceneTest.py

```diff
@@ -1878,5 +1878,129 @@ def testDisplayColor( self ) :
 			vertexPlane
 		)
 
+	def testCanReferenceTags( self ) :
+
+		# Write "tags.usda" via SceneInterface.
+
+		tagsFileName = os.path.join( self.temporaryDirectory(), "tags.usda" )
+		root = IECoreScene.SceneInterface.create( tagsFileName, IECore.IndexedIO.OpenMode.Write )
+		child = root.createChild( "child" )
+		child.writeTags( [ "tagA", "tagB" ] )
+		grandChild = child.createChild( "grandChild" )
+		grandChild.writeTags( [ "tagB", "tagC" ] )
+		del root, child, grandChild
+
+		# Write "test.usda" via USD API, referencing "tags.usda".
+
+		fileName = os.path.join( self.temporaryDirectory(), "test.usda" )
+		root = pxr.Usd.Stage.CreateNew( fileName )
+		root.OverridePrim( "/child" ).GetReferences().AddReference( tagsFileName, "/child" )
+		root.GetRootLayer().Save()
+		del root
+
+		# Load "test.usda" via SceneInterface, and check that we can read the tags.
+
+		root = IECoreScene.SceneInterface.create( fileName, IECore.IndexedIO.OpenMode.Read )
+		child = root.child( "child" )
+		self.assertEqual( { str( x ) for x in child.readTags() }, { "tagA", "tagB" } )
+		grandChild = child.child( "grandChild" )
+		self.assertEqual( { str( x ) for x in grandChild.readTags() }, { "tagB", "tagC" } )
+
+	def testTagSetEquivalence( self ) :
+
+		# Location      Tags               Sets
+		#
+		# /a            tagOne             setOne : { /a, /a/b }
+		#   /b          tagTwo
+		# /c            			       setTwo : { /c/d }
```

Comment:
FYi, the tabs here make the alignment fail on 8-space-per-tab UIs like GitHub

Reply:
Thanks - fixed in 2a8ecce.